README.md

```diff
@@ -2,72 +2,84 @@
 
 ## Overview
 
-In this lesson you're going to build an application that provides a Command Line Interface (CLI) to an external data source. The CLI will be composed of an Object Oriented Ruby application. 
+Congrats! It’s time to work on your first project. In this lesson you're going to build a [Ruby gem](https://guides.rubygems.org/what-is-a-gem/) that provides a Command Line Interface (CLI) to an external data source. The CLI will be composed of an Object Oriented Ruby application. You'll use all you've learned about Ruby and [CLI Applications in Ruby](https://learn.co/tracks/full-stack-web-development-v5/intro-to-ruby-development/command-line-applications/cli-applications-in-ruby) to complete this project.
 
-**Please note that while you will be writing code to scrape data from a public website, what we're really looking for is your capacity to effectively write good object oriented ruby code (objects, not hashes, separation of concerns, etc.) - we're less interested in the details of the regex's or selector you're using to parse the web pages that you're scraping.**
+**Please note that while you will be writing code to [scrape data](https://learn.co/tracks/full-stack-web-development-v5/object-oriented-ruby/scraping/scraping) or utilize an API from a public website, we are really looking for you to demonstrate your ability  to write concise, [easy-to-read](https://robots.thoughtbot.com/what-is-good-code) Object-Oriented Ruby code (e.g. objects, not hashes; separation of concerns). The details of the regex's or the selectors you use to parse your scraped web pages are less important.**
```

Comment:
Now that I think about it, I might remove the thoughtbot link as it talks a lot about TDD, which I don't think we are going to recommend to any new student.

I do like the links to our curriculum though!